package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kinesis-streams",
-  "version": "0.11.0",
+  "version": "0.12.0",
```

Comment:
NBD but I try to keep version changes separate from code changes. Ultimately, it's the git tag that's the authority.

Reply:
fair point, I can remove this change